.github/workflows/perf-v2.js.yml

```diff
@@ -0,0 +1,52 @@
+name: Perf Test V2
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  benchmark:
+    name: Performance regression check
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+
+      # Clear out potential user data dirs left over from previous failures.
+      - run: rm -rf /tmp/replicache-playwright-*
+      - run: npm ci
+      - run: npx playwright install-deps
+      - run: npm run build --if-present
+
+      # Run benchmark and stores the output to a file
+      - name: Run benchmark
+        run: node perf/runner.js --format=json | tee perf-v2-output.json
+
+      # Download previous benchmark result from cache (if exists)
+      - name: Download previous benchmark data
+        uses: actions/cache@v1
+        with:
+          path: ./cache
+          key: ${{ runner.os }}-benchmark
+
+      # Run `github-action-benchmark` action
+      - name: Store benchmark result
+        uses: rhysd/github-action-benchmark@v1
+        with:
+          # What benchmark tool the output.json came from
+          tool: 'customSmallerIsBetter'
```

Comment:
Nice. I didn't think of not outputting a benchmarkjs formatted file.

Reply:
Thanks for the review and hints on testing.